### PR TITLE
V20

### PR DESCRIPTION
**Pull request by kajes**

### CODE REVIEW

File: appgate/resource_appgate_appliance.go [hunk 1]
```diff
@@ -1033,6 +1043,11 @@ func resourceAppgateAppliance() *schema.Resource {
 											},
 										},
 									},
+									"labels_disabled": {
+										Type:     schema.TypeList,
+										Optional: true,
+										Elem:     schema.TypeString,
+									},
```
Comment (mandopaloooza) [inline appgate/resource_appgate_appliance.go:1050]: Running this in 6.1 and 6.2 yields this error. We support 2 versions back so most likely missing a version checker in the code that touches this resource
```
=== NAME  TestAccAppliance62MetricsAggregator
    resource_appgate_appliance_test.go:4707: Step 1/2 error: Error running pre-apply refresh: exit status 1

        Error: metrics_aggregator.0.prometheus_exporter.0.labels_disabled: unknown element type (internal)

          with appgatesdp_appliance.appliancev62_metrics_aggregator,
          on terraform_plugin_test.tf line 6, in resource "appgatesdp_appliance" "appliancev62_metrics_aggregator":
           6: resource "appgatesdp_appliance" "appliancev62_metrics_aggregator" {
```

File: appgate/resource_appgate_site.go [hunk 2]
```diff
@@ -302,6 +297,10 @@ func resourceAppgateSite() *schema.Resource {
 										Optional: true,
 										Default:  true,
 									},
+									"partition": {
+										Type:     schema.TypeString,
+										Optional: true,
+									},
```
Comment (mandopaloooza) [inline appgate/resource_appgate_site.go:303]: Need to increment the number of elements in test
```
=== RUN   TestAccSiteBasicAwsResolverWithoutSecret
    resource_appgate_site_test.go:615: Step 1/2 error: Check failed: Check 13/39 error: appgatesdp_site.test_site: Attribute 'name_resolution.0.aws_resolvers.0.%' expected "11", got "12"
```

**Comment by kajes** [comment]
Test are working now, so proper code review in place

**Review by mandopaloooza — CHANGES_REQUESTED** [review]
Looks good overall + Test passes on 6.1 - 6.3

I know it is tedious work but markdown files (documentation) needs updating for some of the new/old fields. From a quick scan with my eyes, I think we are missing the following fields:
- `prometheus_exporter.labels_disabled`
-  `azure_monitor.scope`
-  fields for datadog and corelogix

**Review by mandopaloooza — APPROVED** [review]

**Pull request merged**